File: ServiceBus/MessagingPython/Queue/receive-message.py
from azure.servicebus import ServiceBusClient

connection_str = ""
queue_name = "queue001"

def receive_messages(connection_str, queue_name):
    servicebus_client = ServiceBusClient.from_connection_string(connection_str)
    receiver = servicebus_client.get_queue_receiver(queue_name)
    with receiver:
        for message in receiver:
            # Do something with the message
            print(message)
            receiver.complete_message(message)
    receiver.close()
    servicebus_client.close()

receive_messages(connection_str, queue_name)

File: ServiceBus/MessagingExpress/Queue/receive-message.js
const {ServiceBusClient} = require("@azure/service-bus");
const connectionString = ""

const queueName = "queue001";
const sbClient = new ServiceBusClient(connectionString);
const receiver = sbClient.createReceiver(queueName);

async function receiveMessage(){

    // subscribe to messages from queue
    receiver.subscribe({
        processMessage: async (message) => {
            //process each message
            console.log(message.body);
        },
        processError: async (err) => {
            console.log(err);
        }
    })
}

receiveMessage()
    .then(() => {
        return receiver.close();
    })
    .then(() => {
        return sbClient.close();
    })
    .catch((err) => {
        console.error(err);
    });